geoff/.htaccess

```diff
@@ -0,0 +1,12 @@
+AddType application/rdf+xml .rdf
+AddType text/turtle .ttl
+
+Header set Access-Control-Allow-Origin *
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^/?$ http://theme-e.adaptcentre.ie/geoff/ [R=302,L]
```

Comment:
Looks like this rule will take precedence over the below rules.  You probably want to put it at the bottom so the rdf/ttl rules are checked first.